src/lib/preprocess.js

```diff
@@ -20,11 +20,4 @@ function render(source, config) {
   const configWithFrontmatter = addFrontmatter(ast, config)
   const content = markdoc.transform(ast, configWithFrontmatter)
   return markdoc.renderers.html(content)
```

Comment:
Maybe instead of using Markdoc's renderer, it can reuse `Markdoc.svelte` here?
That way tags can work for the preprocessor approach too.

The preprocessor can use the config to figure out what tags are supported.

Reply:
Yeah great idea! I've not looked into the preprocessor API before but it looks like this kind of thing should be possible?

Reply:
Yes, all the preprocessor needs to do is return a string containing valid svelte code. I think it should be doable.